.github/workflows/pkg.yml

```diff
@@ -0,0 +1,29 @@
+name: Packaging
+
+on:
+  # Since this workflow is quite costly, we only run it automatically on
+  # release tags. Otherwise trigger it manually.
+  push:
+    tags:
+      - v*
+
+  # Allow manually triggering the workflow.
+  workflow_dispatch:
+
+jobs:
+  package:
+    uses: NLnetLabs/ploutos/.github/workflows/pkg-rust.yml@v7
+    secrets:
+      DOCKER_HUB_ID: ${{ secrets.DOCKER_HUB_ID }}
```

Comment:
TODO: The corresponding secrets for this (our Docker Hub automation user id and a newly generated access token granting access to a new dnsi Docker Hub repo) need to be provisioned in GH project settings.